worldengine/draw.py

```diff
@@ -9,6 +10,36 @@
 # Helper values
 # -------------
 
+### For draw_satellite ###
+NOISE_RANGE = 10 # a random value between -NOISE_RANGE and NOISE_RANGE will be added to the rgb of each pixel
+
+# These are arbitrarily-chosen elevation cutoffs for 4 different height levels. 
+# Some color modifiers will be applied at each level
+HIGH_MOUNTAIN_ELEV = 215
```

Comment:
Take a look at world.py for an explanation of the 3, I think that's where that list is created. It is basically a list of percentages that are you to calculate mountain-heights, hill-heights etc.

In your case, since you are working with normalized values, you could bypass calling find_threshold() and instead manually calculate the necessary value with the line of code I gave above.